packages/x-components/src/wiring/wiring.types.ts

```diff
@@ -137,15 +130,15 @@ export interface TimedWireOperatorOptions {
  *
  * @public
  */
-export interface WireService<SomeService> {
+export interface WireService<SomeService extends Record<string, MonadicFunction>> {
```

Comment:
What I'm trying to do here is assert that the passed service has functions with one or less parameters. Then extracting the key is as easy as doing `keyof SomeService`.